packages/sanity/src/desk/deskTool.ts

```diff
@@ -119,7 +119,11 @@ export const deskTool = definePlugin<DeskToolOptions | void>((options) => {
         getIntentState,
         // Controlled by sanity/src/desk/components/deskTool/DeskTitle.tsx
         controlsDocumentTitle: true,
-        options,
+        options: {
+          ...options,
```

Comment:
My only concern here is that options takes any as prop.
And this spread will return you something you won't be expecting.
Is it possible to update the options type to something like:

```
interface Options = {
 [key:string]: any;
 /**
 * @internal
 * /
 __internal_toolMenuButtonWrapper?: React.ReactNode
}
```

Reply:
[discussed this topic in a meeting]